app/src/components/template/Dashboard.js

```diff
@@ -33,7 +33,11 @@ const Dashboard = () => {
     return (
         <div className={classes.root}>
             <CssBaseline />
-            <AppBar position={'absolute'} className={clsx(classes.appBar, open && classes.appBarShift)} theme={OBStyles.themeBackground}>
+            <AppBar 
+                position={'absolute'}
+                className={clsx(classes.appBar, open && classes.appBarShift)}

```

Comment:
Just to keep our code style consistent, you should use classNames:
```suggestion
                className={classNames(classes.appBar, open && classes.appBarShift)}
```